zilliqa/tests/it/main.rs

```diff
@@ -601,7 +605,14 @@ impl Network {
                 } else {
                     self.nodes.iter().collect()
                 };
-                for (index, node) in nodes.iter().enumerate() {
+
+                for node in &nodes {
```

Comment:
Btw, this was incorrect in the last PR to master; you can't use the iterated index because then it always seems like the message is going to node 0, when it isn't.